extension/ts/services/request.ts

```diff
@@ -0,0 +1,7 @@
+import * as cheerio from "cheerio";
+
+const get = (url: string) => fetch(url).then((res) => res.text());
+
+const get$ = (url: string) => get(url).then((text) => cheerio.load(text));
+
+export {get, get$};
```

Comment:
why do we export `get` if we don't use it anywhere else

Reply:
It doesn't need to be exported. In my mind this was a Request API that may or may not be used in the future